docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.7"
+services:
+  nginx:
+    build:
+      context: infra/nginx
+      dockerfile: Dockerfile
+    container_name: nginx
+    ports:
+      - target: 80
+        published: 8080
+        protocol: tcp
+        mode: host
+    links:
+      - vvgo
+      - minio
+    networks:
+      - frontend
+      - backend
+
+  vvgo:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: vvgo
+    environment:
+      MINIO_ENDPOINT: minio:9000
+      MINIO_ACCESS_KEY_ID: minioadmin
+      MINIO_SECRET_ACCESS_KEY: minioadmin
+    expose:
+      - 8080
+    links:
+      - minio
+    networks:
+      - backend
+
+  minio:
+    build:
+      context: infra/minio
+      dockerfile: Dockerfile
+    container_name: minio
+    environment:
+      MINIO_REGION_NAME: us-east-1
+      MINIO_CACHE_EXPIRY: 1
+      MINIO_CACHE_QUOTA: 99
+      MINIO_CACHE_AFTER: 1
+    expose:
+      - 9000
```

Comment:
Why are we exposing Minio directly, and not through a reverse proxy?

Reply:
It's exposed on the background network so that vvgo.org and hit it.

Reply:
Whoops. Shouldn't do code review while tired.